### PR TITLE
Comments on DM-6256

### PR DESCRIPTION
**Pull request by jdswinbank**
Not a full "review", but catching minor typos etc as I read.

### CODE REVIEW

File: sections/data_release_production_pipelines.tex [hunk 1]
```diff
@@ -404,22 +398,160 @@ \subsubsection{FinalJointCal}
...
+\begin{description}
+\item[CalExp.\{Image,Background,Mask\}]  See above.
+\item[DIAExp] A CCD-level \hyperref[sec:spImagesExposure]{Exposure} that is the difference between the CalExp and a template coadd, in the coordinate system and with the PSF of the CalExp.
+\item[DIASource] A \hyperref[sec:spTablesSource]{SourceCatalog} containing sources detected and measured on the DIAExp images.
+\item[ConstantPSFCoadd] A coadd data product (\hyperref[sec:spImagesExposure]{Exposure} or subclass thereof) with a constant, predefined PSF.
+\item[DeepCoadd] A coadd data product built to optimize depth at the possible expense of seeing.
+\item[BestSeeingCoadd] A coadd data product built to optimize image quality at the possible expense of depth.  Depending on the algorithm used, this may be the same as DeepCoadd.
+\item[ShortPeriodCoadd] A coadd data product built from exposures in a short range of epochs, such as a year, rather than the full survey.
+\item[LikelihoodCoadd] A coadd formed by correlating each image with its own PSF before combining them, used for detection and possibly building other coadds.
+\item[ShortPeriodLikelihoodCoad] Short-period likelihood coadds will also be built.
```
Comment (jdswinbank) [inline sections/data_release_production_pipelines.tex]: s/Coad/Coadd/

File: sections/data_release_production_pipelines.tex [hunk 2]
```diff
@@ -404,22 +398,160 @@ \subsubsection{FinalJointCal}
...
+\item[TemplateCoadd] A coadd data product used for difference imaging in both DRP and AP, \emph{or} some other tract/patch/filter data product from which the template for a given airmass can be derived.
+\end{description}
+
+The nature of these coadd data products depends critically on whether we are able to develop efficient algorithms for optimal coaddition, and whether these coadds are suitable for difference imaging.  These algorithms are mathematically well-defined but computationally difficult; see \href{http://dmtn-015.lsst.io/en/latest/}{DMTN-15} for more information.  We will refer to the coadds produced by these algorithms as ``decorrelated coadds''; a variant with constant PSF (``constant-PSF partially decorrelatd coadd'') is also possible.  This choice is also mixed with the question of how we will correct for differential chromatic refraction in difference imaging; some algorithms for DCR correction involve templates that are the result of inference on input exposures rather than coaddition.  There are five main scenarios for our use of decorrelated coadds:
```
Comment (jdswinbank) [inline sections/data_release_production_pipelines.tex]: s/decorrelatd/decorrelated/

File: sections/data_release_production_pipelines.tex [hunk 3]
```diff
@@ -404,22 +398,160 @@ \subsubsection{FinalJointCal}
...
+\begin{description}
+  \item[A] \label{option:decorrelated-everything} We use decorrelated coadds for all final coadd products.  DeepCoadd and ShortPeriodCoadd will be standard decorrelated coadds with a spatially-varying PSF, and ConstantPSFCoadd and TemplateCoadd will be constant-PSF partially-decorrelated coadds.  The BestSeeingCoadd data product will be dropped, as it will be redundant with DeepCoadd.  This will make coadds more expensive and complex to build, and require more algorithm development for coaddition, but will improve coadd-based measurements, make it easier to warm-start multi-epoch measurements.  Difference imaging may be easier, and more visits may be usable as inputs to templates due to softened or eliminated seeing cut.
```
Comment (jdswinbank) [inline sections/data_release_production_pipelines.tex]: s/make/and make/

File: sections/data_release_production_pipelines.tex [hunk 4]
```diff
@@ -404,22 +398,160 @@ \subsubsection{FinalJointCal}
...
+
+Further differences in the pipelines themselves due to the presence or absence of decorrelated coadds will be described in the sections below.
 
 \subsubsection{WarpAndPsfMatch}
 \label{sec:drpWarpAndPsfMatch}
+
+This pipeline resamples and then PSF-matches CalExp images from a visit into a single patch-level image with a constant PSF.  The resampling and PSF-matching can probably be accomplished separately by delegating to the \hyperref[sec:acWarping][Image Warping] and \hyperref[sec:acPSFHomogenization]{PSF Homogenization} algorithmic components, respectively.  These operations can also be performed in the opposite order if the matched-to PSF is first transformed to the CalExp coordinate systems (so subsequent resampling yields a constant PSF in the coadd coordinate system).  Doing PSF-matching first may be necessary (or at least easier to implement) for undersampled images.
```
Comment (jdswinbank) [inline sections/data_release_production_pipelines.tex]: s/[Image Warping]/{Image Warping}/

File: sections/data_release_production_pipelines.tex [hunk 5]
```diff
@@ -404,22 +398,160 @@ \subsubsection{FinalJointCal}
...
+
+The outputs of BackgroundMarchAndReject are updated Background and Mask components for the CalExp product.  Because it is not built with the final photometric and astrometric calibration, the PSF-matched coadd built here is discarded.
+
+\subsubsection{WarpTemplates}
+\label{sec:drpWarpTemplates}
+
+This pipeline is responsible for generating the resampled visit-level images (TemplateWarp) used to build template coadds for difference imaging.  The algorithmic content of this pipeline and the nature of its outputs depends on whether we are using decorrelated coadds (\ref{option:decorrelated-everything} at the beginning of \ref{sec:drp_coaddition_and_diffim}), PSF-matched coadds (\ref{option:psf-matched-templates-b} or \ref{option:psf-matched-templates-c}), or inferring templates (\ref{option:inferred-templates-b} or \ref{option:inferred-templates-c}).
```
Comment (jdswinbank) [inline sections/data_release_production_pipelines.tex]: These references all resolve to the section number, so we get "decorrelated coadds (5.2 at the beginning of 5.2), PSF-matched coadds (5.2 or 5.2), or inferring templates (5.2 or 5.2)."

Comment (TallJimbo) [inline sections/data_release_production_pipelines.tex]: Fixed in my most recent push.

File: sections/data_release_production_pipelines.tex (hunk 4 above)
Comment (TallJimbo) [inline sections/data_release_production_pipelines.tex]: Fixed.

File: sections/data_release_production_pipelines.tex (hunk 3 above)
Comment (TallJimbo) [inline sections/data_release_production_pipelines.tex]: Fixed.

File: sections/data_release_production_pipelines.tex (hunk 2 above)
Comment (TallJimbo) [inline sections/data_release_production_pipelines.tex]: Fixed.

**Pull request merged**